src/components/userCars/car/car.jsx

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import { Card } from "antd";
+import {
+    FormOutlined,
+    SnippetsOutlined,
+    ContactsOutlined,
+    ColumnWidthOutlined
+} from "@ant-design/icons";
+import { MdBrush, MdStickyNote2, MdList } from "react-icons/md";
+import Text from "antd/es/typography/Text";
+import bucket_icon from "../../../assets/images/cars/bucket.svg";
+import heavy_icon from "../../../assets/images/cars/heavy.svg";
+import apps_icon from "../../../assets/images/cars/apps.svg";
+import info_icon from "../../../assets/images/cars/info.svg";
+import insurance_icon from "../../../assets/images/cars/insurance.svg";
+import passport_icon from "../../../assets/images/cars/passport.svg";
+
+function Car(data) {
+    console.log(data);
```

Comment:
Why do we need this information in console?

Reply:
I forgot to delete